### ISSUE TITLE
OCMOD issues

### ISSUE THREAD

**Issue by huba80** [condor2/Opencart_2302#15]
Hi,

I tried to install 2 different extensions. 
One says "invalid file type" for "webpimageconverterv1.4.6.ocmod.zip" the other is Nitrocache IO extension from here:

https://nitropack.io/platform/opencart

Nitrocache gives the following error:

```
[26-Apr-2021 07:36:16 UTC] PHP Fatal error:  Uncaught Exception: Error: Table 'tolokapu_zednewpavo.oc_extension_install' doesn't exist<br />Error No: 1146<br />INSERT INTO `oc_extension_install` SET `filename` = '2.3.x.nitropackio.ocmod.zip', `extension_download_id` = '0', `date_added` = NOW() in /home/tolokapu/public_html/system/library/db/mysqli.php:48
Stack trace:
#0 /home/tolokapu/public_html/system/library/db.php(45): DB\MySQLi->query('INSERT INTO `oc...')
#1 /home/tolokapu/public_html/admin/model/extension/extension.php(29): DB->query('INSERT INTO `oc...')
#2 /home/tolokapu/public_html/system/storage/modification/system/engine/loader.php(178): ModelExtensionExtension->addExtensionInstall('2.3.x.nitropack...')
#3 /home/tolokapu/public_html/system/engine/proxy.php(25): Loader->{closure}(Array, Array)
#4 /home/tolokapu/public_html/admin/controller/extension/installer.php(185): Proxy->__call('addExtensionIns...', Array)
#5 /home/tolokapu/public_html/system/storage/modification/system/engine/action.php(5 in /home/tolokapu/public_html/system/library/db/mysqli.php on line 48
```

Both extensions were working on another site.

**Comment by condor2** [comment]
If was a clean install with this version should work.
If you replaced a clean version of 2.3.0.2 with this one you need to patch database.
Check Readme -> Ocmod installer

**Comment by huba80** [comment]
Thanks. Ive overcome that problem but now Im having a new problem. I wanted to change a few store settings and when I go to System/Settings? Store I get a blank page. Error log:

``2021-05-03 8:26:19 - PHP Notice:  date_default_timezone_set(): Timezone ID '' is invalid in /home/tolokapu/public_html/admin/controller/setting/setting.php on line 530``

		`date_default_timezone_set($this->config->get('config_timezone'));`

It turns out that there is no config_timezone in key column in phpmyadmin.

I patched my old 2.3.0.2  with your version. I ran the following SQL query:

```
INSERT INTO `oc_setting` (`store_id`, `code`, `key`, `value`, `serialized`) VALUES
(0, 'config', 'config_timezone', 'UTC', 0);

Now it seems to be working but Im not sure what else could be missing from Settings table.
```
